deps.edn

```diff
@@ -87,55 +87,55 @@
-  org.clojure/core.memoize                  {:mvn/version "1.0.250"}            ; useful FIFO, LRU, etc. caching mechanisms
+  org.clojure/core.memoize                  {:mvn/version "1.0.253"}            ; useful FIFO, LRU, etc. caching mechanisms
   org.clojure/data.csv                      {:mvn/version "1.0.0"}              ; CSV parsing / generation
   org.clojure/java.classpath                {:mvn/version "1.0.0"}              ; examine the Java classpath from Clojure programs
   org.clojure/java.jdbc                     {:mvn/version "0.7.12"}             ; basic JDBC access from Clojure
   org.clojure/java.jmx                      {:mvn/version "1.0.0"}              ; JMX bean library, for exporting diagnostic info
   org.clojure/math.combinatorics            {:mvn/version "0.1.6"}              ; combinatorics functions
-  org.clojure/math.numeric-tower            {:mvn/version "0.0.4"}              ; math functions like `ceil`
-  org.clojure/tools.logging                 {:mvn/version "1.1.0"}              ; logging framework
-  org.clojure/tools.namespace               {:mvn/version "1.1.0"}
+  org.clojure/math.numeric-tower            {:mvn/version "0.0.5"}              ; math functions like `ceil`
+  org.clojure/tools.logging                 {:mvn/version "1.2.4"}              ; logging framework
+  org.clojure/tools.namespace               {:mvn/version "1.2.0"}
   org.clojure/tools.reader                  {:mvn/version "1.3.6"}
   org.clojure/tools.trace                   {:mvn/version "0.7.11"}             ; function tracing
-  org.eclipse.jetty/jetty-server            {:mvn/version "9.4.43.v20210629"}   ; web server
-  org.flatland/ordered                      {:mvn/version "1.5.9"}              ; ordered maps & sets
-  org.graalvm.js/js                         {:mvn/version "21.2.0"}           ; JavaScript engine
+  org.eclipse.jetty/jetty-server            {:mvn/version "9.4.44.v20210927"}   ; web server
+  org.flatland/ordered                      {:mvn/version "1.15.10"}            ; ordered maps & sets
+  org.graalvm.js/js                         {:mvn/version "22.0.0"}             ; JavaScript engine
   org.liquibase/liquibase-core              {:mvn/version "3.6.3"               ; migration management (Java lib)
                                              :exclusions  [ch.qos.logback/logback-classic]}
-  org.mariadb.jdbc/mariadb-java-client      {:mvn/version "2.6.2"}              ; MySQL/MariaDB driver
-  org.postgresql/postgresql                 {:mvn/version "42.2.23"}            ; Postgres driver
-  org.slf4j/slf4j-api                       {:mvn/version "1.7.32"}             ; abstraction for logging frameworks -- allows end user to plug in desired logging framework at deployment time
+  org.mariadb.jdbc/mariadb-java-client      {:mvn/version "2.7.5"}              ; MySQL/MariaDB driver
+  org.postgresql/postgresql                 {:mvn/version "42.3.1"}             ; Postgres driver
```

Comment:
Probably wanna get PG 42.3.2 while we're at it, considering the new connection string stuff
https://github.com/pgjdbc/pgjdbc/blob/master/CHANGELOG.md

Reply:
done